<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-2S7M9HQ" ignored="1" name="Application Test" tests="3" failures="3" timestamp="2020-08-01T13:08:51 IST" time="168.739" errors="0">
  <testcase name="testLogin" time="22.505" classname="com.PracticeTestClasses.TestClasses.LoginTests">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.PracticePageClasses.Pages.NavigationPage.verifyHeader(NavigationPage.java:55)
at com.PracticeTestClasses.TestClasses.LoginTests.testLogin(LoginTests.java:32)
at java.util.ArrayList.forEach(ArrayList.java:1257)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- testLogin -->
  <testcase name="filterByCategory" time="58.451" classname="com.PracticeTestClasses.TestClasses.AllCoursesTest">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.PracticePageClasses.Pages.CategoryFilterPage.findCoursesCount(CategoryFilterPage.java:41)
at com.PracticeTestClasses.TestClasses.AllCoursesTest.filterByCategory(AllCoursesTest.java:38)
at java.util.ArrayList.forEach(ArrayList.java:1257)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- filterByCategory -->
  <testcase name="verifySearchCourse" time="13.077" classname="com.PracticeTestClasses.TestClasses.AllCoursesTest">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.PracticeTestClasses.TestClasses.AllCoursesTest.verifySearchCourse(AllCoursesTest.java:30)
at java.util.ArrayList.forEach(ArrayList.java:1257)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- verifySearchCourse -->
  <testcase name="testInvalidLogin" time="0.0" classname="com.PracticeTestClasses.TestClasses.LoginTests">
    <ignored/>
  </testcase> <!-- testInvalidLogin -->
</testsuite> <!-- Application Test -->
